src/main/java/org/eclipse/uprotocol/cloudevent/README.adoc

```diff
@@ -11,8 +11,93 @@ https://github.com/eclipse-uprotocol/uprotocol-spec/blob/main/up-l1/cloudevents.
 === CloudEventFactory
 Factory class that builds the various types of CloudEvents for uProtocol (publish, notification, request, response)
 
+
 == Examples
 
-_Coming soon_, for now please refer to the test folder for examples of how to use CloudEventFactory
+=== Building an uuri
+[source,java]
+----
+      UEntity use = UEntity.newBuilder().setName("body.access").build();
+      UResource res= (UResource.newBuilder().setName("door").setInstance("front_left").setMessage("Door").build();
+      UUri uri = UUri.newBuilder().setEntity(use).setResource(res).build();
+      String source = LongUriSerializer.instance().serialize(uri);
+----
+
+=== Build proto payload
```

Comment:
We want people to use the factory and not build cloudevents using the io.cloudevents.v1.proto.CloudEvent....

Reply:
This is "Any" proto payload required for CloudEvent factory methods.